sqlair.go

```diff
@@ -163,3 +164,61 @@ func (q *Query) One(outputArgs ...any) error {
 	iter.Decode(outputArgs...)
 	return iter.Close()
 }
+
+// All iterates over the query and decodes all the rows.
+// resultColumns must contain pointers to slices of each of the query arguments e.g.
+// > q.All(&[]*Person{}, &[]*Address{}) or q.All(&[]Address{}, &[]Person{})
+// It fabricates all structs as needed.
+func (q *Query) All(slicePointers ...any) error {
```

Comment:
I'd stick to the eariler theme and call them `sliceArgs` (we don't call them `outputPointers`, or `scanPointers`).

Reply:
Changed.